doc/release-notes/0.23/0.23.md

```diff
@@ -74,6 +74,11 @@ The following table covers notable changes in v0.23.0. Further information about
 <td valign="top">For compatibility with the reference implementation, OpenJ9 now supports this option.</td>
 </tr>
 
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/issues/6914">#6914</a></td>
+<td valign="top">JITServer technology preview extended to Linux on Power and Linux on Z</td>
+<td valign="top">OpenJDK8 and 11 (Linux x86-64, Linux on Power and Linux on Z only)</td>
+<td valign="top">JITServer decouples the JIT compiler from the OpenJ9 VM, freeing up CPU and memory for an application. JITServer then runs in its own process, either locally or on a remote machine where resources can be separately managed..</td>
```

Comment:
I would add a further comma after "remote machine" as this part of the sentence seems to be a "drop-in" clause. Also remove the extra full stop at the end, which is always hard to spot in HTML edit.

We'll need to make some corresponding changes to the user documentation, both in this topic https://eclipse.github.io/openj9-docs/version0.23/ and https://eclipse.github.io/openj9-docs/jitserver/. I'll start those changes.

Reply:
@mpirvu - Are we supporting z31 for OpenJDK 8 or is this 64-bit only?

Reply:
I updated the text with the suggested changes.

> Are we supporting z31 for OpenJDK 8 or is this 64-bit only

We only test 64-bit platforms, so Linux-31 bit is not supported, though it may work.